### DISCUSSION TITLE
Error when requesting cert with a SAN, not sure if bug or misconfiguration my end

### DISCUSSION THREAD

**Discussion by garethterrace** [polymind-inc/acmebot#694] [Q&A] [answered]
I'm migrating away from another letsencrypt site extension and so far everything's going well.

We've got some certificates with SANs:
app1.example.com and app.example.com

When I add them to the dashboard I get the following error: 

>  Order includes different number of names than CSR specifies","ProblemStatus":403,"Message":"Error finalizing order :: Order includes different number of names than CSR specifies

When I request a single domain, all works well though.

Any help appreciated!

**Comment by shibayan** [comment]
Does a certificate for the domain of the same name already exist in Key Vault?

Because Acmebot uses the first domain name of the SAN to generate the name of the Key Vault certificate, there may be conflicts depending on how the SAN is issued.

**Reply by garethterrace** [reply]
not in keyvault for that specific subdomain but we do have one for a different subdomain

at the moment I just have test.example.com in the keyvault. I wonder if that's the issue. Maybe one cert for all subdomains then?

**Reply by garethterrace** [reply]
just tried again requesting all of the subdomains in one cert with an empty keyvault and got the same error

**Reply by garethterrace** [reply, marked as answer]
Sorted it - I purged all the certs from the store and retried, now it's working fine. Not sure if the purge helped, but it's sorted. 

Sidenote: This is lovely to use - thanks so much for your efforts here
